src/passes/Print.cpp

```diff
@@ -2014,7 +2014,13 @@ struct PrintExpressionContents
   }
   void visitCallRef(CallRef* curr) {
     if (curr->isReturn) {
-      printMedium(o, "return_call_ref");
+      if (printUnreachableReplacement(curr->target)) {
```

Comment:
return calls are always unreachable aren't they? this would always happen then, unless I'm missing something.

Reply:
We're passing `curr->target`, so we only print the replacement if the _target_ is unreachable.